src/Raven.Client/Http/RequestExecutor.cs

```diff
@@ -1743,6 +1746,12 @@ private sealed class BroadcastState<TResult>
             public IDisposable ReturnContext;
         }
 
+        public enum CommandUnsuccessfulResponseBehavior
+        {
+            None,
+            WrapException
+        }
```

Comment:
internal and maybe let's move this to `AbstractCommandResponseBehavior` ?

Reply:
Done

Reply:
It's still `public`, isn't it?

Reply:
I believe it is okay, since now it is under an internal class

Reply:
OK. Merging